app/models/release.rb

```diff
@@ -44,6 +44,9 @@ def self.find_by_version!(version)
   private
 
   def assign_release_number
+    default_number = 1
```

Comment:
Not sure what you mean exactly, can you elaborate a bit?  The default_number is actually refering to the fact that Release.number is set to default=1 in database, so we give it a variable name to make our code more readable.